lib/utils.js

```diff
@@ -669,3 +671,64 @@ function autoEnd (stream, eventName) {
     stream.flushSync()
   }
 }
+
+/**
+ * Returns a object which will be used to handle custom levels
+ *
+ * @param {string|object} cLevels Custom Levels can be object or string
+ *
+ * @returns {object} A object like level: level_name
+ */
+function handleCustomlevelsOpts (cLevels) {
+  if (!cLevels) return {}
+
+  if (typeof cLevels === 'string') {
+    return cLevels
+      .split(',')
+      .reduce((agg, value, idx) => {
+        const [levelName, levelIdx = idx] = value.split(':')
+        agg[levelIdx] = levelName.toUpperCase()
+        return agg
+      },
+      { default: 'USERLVL' })
+  } else if (typeof cLevels === 'object') {
```

Comment:
An instance of `[]` would pass this check. The correct check is:
```suggestion
  } else if (Object.prototype.toString.call(cLevels) === '[object Object]') {
```

Reply:
True, i never did like that, i will change both checks, thank you.